test/periphery/integration/merkle-campaign/instant/claim/claim.t.sol

```diff
@@ -1,106 +1,21 @@
 // SPDX-License-Identifier: UNLICENSED
 pragma solidity >=0.8.22 <0.9.0;
 
-import { Errors } from "src/periphery/libraries/Errors.sol";
+import { ISablierMerkleBase } from "src/periphery/interfaces/ISablierMerkleBase.sol";
 
-import { MerkleCampaign_Integration_Test } from "../../MerkleCampaign.t.sol";
+import { Claim_Integration_Test } from "../../shared/claim/claim.t.sol";
 
-contract Claim_MerkleInstant_Integration_Test is MerkleCampaign_Integration_Test {
-    function test_RevertGiven_CampaignExpired() external {
-        uint40 expiration = defaults.EXPIRATION();
-        uint256 warpTime = expiration + 1 seconds;
-        bytes32[] memory merkleProof;
-        vm.warp({ newTimestamp: warpTime });
-        vm.expectRevert(abi.encodeWithSelector(Errors.SablierMerkleBase_CampaignExpired.selector, warpTime, expiration));
-        merkleInstant.claim({ index: 1, recipient: users.recipient1, amount: 1, merkleProof: merkleProof });
+contract Claim_MerkleInstant_Integration_Test is Claim_Integration_Test {
```

Comment:
How about also moving it to `MerkleInstant.t.sol` file? The setUp function can then be re-used from `MerkleInstant_Integration_Shared_Test`.

Reply:
Added this suggestion in https://github.com/sablier-labs/v2-core/pull/1016.